source/CardGames.py

```diff
@@ -224,15 +207,31 @@ def start_game(self):
     dealer.dealCards(5, self.players)
     for player in self.players:
       player.showHand(player.hand)
-    
+
+  def check_if_empty(self):
+    if len(self.info_player_asked) == 0:
+          self.game_over()
+
+  def game_over(self):
+    j = 0
+    gm = ["G", "A", "M", "E", "O", "V", "E","R"]
+    for i in range(20):
+      print(" "*j, ".")
+      j += 3
+      if i == 10:
+        for i in range (len(gm)):
+          print(" "*j, gm[i])
+          j += 3
+
+    sys.exit()
```

Comment:
The code looks good! not sure if sys exit will just close the screen, but if it does I think it would be better to let the players be able to look at the screen with the game over screen rather than just closing it, i may be wrong though about the sys exit().

Reply:
Yeah, thanks for the feedback! It doesn't close the screen but it ends the program so that it will no longer ask for input. Once we have everything else figured out, like the winners, then maybe I will include the winner in the game over screen.